app/MMB/src/scenes/tripServices/transportation/TransportationMap.js

```diff
@@ -138,46 +135,15 @@ export class TransportationMap extends React.Component<Props, State> {
     }
   }
 
-  openLocationPicker = async () => {
-    const { currentLocation } = this.state;
-    let currentAddress;
+  shouldComponentUpdate = (nextProps: Props, nextState: State) => {
```

Comment:
this seems abundant. When I comment it, it has just as many rerenders as when it is there